packages/renderer/index.js

```diff
@@ -2,12 +2,23 @@
 
 var events = require('events');
 
+var express = require('express');
 var mocks = require('node-mocks-http');
 
 var createMiddleware = require('hops-middleware');
 
-module.exports = function createRenderer (webpackConfig, watchOptions) {
-  var middleware = createMiddleware(webpackConfig, watchOptions);
+module.exports = function createRenderer (options) {
+  var hopsConfig = options.hopsConfig || {};
```

Comment:
If `hopsConfig` was undefined or falsey, it would already break in https://github.com/xing/hops/pull/269/files#diff-1951c43c9dcbe1f7c6ae7bbb666ef319R24 wouldn't it?

Reply:
Wait, why?

Reply:
If options was falsey it would break, but that is fine for me.